### ISSUE TITLE
Gesture handling causing a null-unsafe operation

### ISSUE THREAD

**Issue by sudha-anecure** [flutter/flutter#71092]
Hi -
I am opening this new ticket on issue #64253, as that issue was not reopened with my comment. We have a repro of that issue.

We got this error from a user using our app built on 1.22.0 version of flutter, on an android device. Here is the stack trace reported on crashlytics from the screen. We have no further information. The stack trace is the only information we have since this is on a production device of an end user.

Error:
```
Null check operator used on a null value
Stack trace:
#0 DragGestureRecognizer.acceptGesture (package:flutter/src/gestures/monodrag.dart:321)
#1 GestureArenaManager.sweep (package:flutter/src/gestures/arena.dart:157)
#2 GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:362)
#3 GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:338)
#4 RendererBinding.dispatchEvent (package:flutter/src/rendering/binding.dart:267)
#5 GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:295)
#6 GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:240)
#7 GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:213)
#8 _rootRunUnary (dart:async/zone.dart:1206)
#9 _CustomZone.runUnary (dart:async/zone.dart:1100)
#10 _CustomZone.runUnaryGuarded (dart:async/zone.dart:1005)
#11 _invoke1 (dart:ui/hooks.dart:265)
#12 _dispatchPointerDataPacket (dart:ui/hooks.dart:174)
Device parameters:
id: PPR1.180610.011
androidId: 4a3352dba7abae8c
board: MT6737
bootloader: unknown
brand: Nokia
device: NE1
display: 00WW_5_18D
fingerprint: Nokia/TA-1032_00WW/NE1:9/PPR1.180610.011/00WW_5_18D:user/release-keys
hardware: mt6735
host: cm-build-42-128
isPhysicalDevice: true
manufacturer: HMD Global
model: TA-1032
product: TA-1032_00WW
tags: release-keys
type: user
versionBaseOs: Nokia/TA-1032_00WW/NE1:9/PPR1.180610.011/00WW_5_180:user/release-keys
versionCodename: REL
versionIncremental: 00WW_5_18D
versionPreviewSdk: 0
versionRelease: 9
versionSdk: 28
versionSecurityPatch: 2020-09-05
```

Can you please investigate a potential fix? Please note that I cannot give you a code with repro as this happens on our production app. Nor can I give you flutter doctor output as we have since the release upgraded our machines to the latest stable version of flutter.

**Comment by darshankawar** [comment]
Will need additional triage, as there's no reproducible use case and the error is coming from production.

**Comment by darshankawar** [comment]
Similar to [64253](https://github.com/flutter/flutter/issues/64253)

**Comment by xu-baolin** [comment]
@sudha-anecure, thanks very much for reporting the issue.
Maybe fixed by #68587, 
but there is no more information, so I can’t be completely sure. Try the get the latest version of Flutter.

**Comment by sudha-anecure** [comment]
Thank you @xu-baolin . We will revisit if we get this issue after the latest version. Please go ahead and close this for now.
